src/globalStyle.js

```diff
@@ -0,0 +1,74 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
```

Comment:
Very inspired of this way of using  a global styling component as a "regular" css file, makes it easier to grasp the concept.